test/python/algorithms/test_iphase_estimator.py

```diff
@@ -0,0 +1,91 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2018, 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Test iterative phase estimation"""
+
+import unittest
+from test.python.algorithms import QiskitAlgorithmsTestCase
+from qiskit.algorithms.phase_estimators import IPhaseEstimation
+import qiskit
+from qiskit.opflow import (H, X, Z)
+
+
+class TestIPhaseEstimation(QiskitAlgorithmsTestCase):
+    """Evolution tests."""
+
+    # pylint: disable=invalid-name
```

Comment:
Could you apply the same updates to these tests as we did in `TestPhaseEstimators`, namely
* default args should not be function calls (but None and then set a default in the test)
* group tests using `@ddt` and `@data`

Reply:
done